rand_distr/src/weighted/alias_method.rs

```diff
@@ -58,8 +58,8 @@ use rand::Rng;
 /// }
 /// ```
 ///
-/// [`WeightedIndex<W>`]: crate::distributions::weighted::alias_method::WeightedIndex
-/// [`Weight`]: crate::distributions::weighted::alias_method::Weight
+/// [`WeightedIndex<W>`]: #
```

Comment:
And can you confirm this link works?

Reply:
TBH, I kind of cheated there :open_mouth:
 Changing `#` to `WeightedIndex` makes the link truly functional (tested locally)